xclim/indices/_agro.py

```diff
@@ -728,7 +728,7 @@ def standardized_precipitation_index(
 
     """
     # "WPM" method doesn't seem to work for gamma or pearson3
-    dist_and_methods = {"gamma": ["ML", "APP"]}
+    dist_and_methods = {"gamma": ["ML", "APP"], "fisk": ["ML"]}
```

Comment:
If you implemented a `_fit_start` for `fisk`, couldn't we allow "APP" here?  Sorry if I forgot a previous discussion about this...

Reply:
Yes, sounds good. The previous idea we discussed was not to use "APP" with `fisk`. But this was back when I was solving saddle point equations with `fsolve` directly in `_fit_start` to get parameters. Now,  there is a simple approximation in terms of mean and variance to get parameters. I agree "APP" should be available for `fisk` now.